packages/web-app-files/src/composables/sort/useSort.ts

```diff
@@ -94,7 +94,7 @@ function createSortDirQueryRef<T>(options: SortOptions<T>): Ref<QueryValue> {
   return useRouteQueryPersisted({
     name: unref(options.sortDirQueryName) || SortConstants.sortDirQueryName,
     defaultValue: unref(firstSortableField(unref(options.fields))?.sortDir),
-    routeName: unref(options.routeName || useRouteName())
+    routeName: unref(options.routeName) || unref(useRouteName())
```

Comment:
afaict these are actual bug fixes, passing in an empty ref should trigger the fallback